tests/docker/Dockerfile

```diff
@@ -140,6 +140,11 @@ COPY --chown=0:0 tests/setup.py /root/tests/
 RUN python3 -m pip install --upgrade --force pip && \
     python3 -m pip install --force --no-cache-dir -e /root/tests/
 
+# Install the OMB tool
+# ToDo: remove checkouting branch when it will be merged to main
+RUN git -C /opt clone https://github.com/redpanda-data/openmessaging-benchmark.git && \
+    cd /opt/openmessaging-benchmark && git checkout ducktape_testing && mvn package
```

Comment:
nit: you can add the branch flag to git clone so you don't have to checkout.
For example: `git -C /opt clone --branch ducktape_testing https://github.com/redpanda-data/openmessaging-benchmark.git`

Reply:
I think this should be a specific `sha1`.